lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -431,9 +510,11 @@ class FatMemManager : public OpSemMemManager {
 FatMemManager::FatMemManager(Bv2OpSem &sem, Bv2OpSemContext &ctx,
                              unsigned ptrSz, unsigned wordSz, bool useLambdas)
     : OpSemMemManager(sem, ctx, ptrSz, wordSz),
-      m_mem(sem, ctx, ptrSz, wordSz, useLambdas) {
+      m_main(sem, ctx, ptrSz, wordSz, useLambdas),
+      m_slot0(sem, ctx, ptrSz, wordSz, useLambdas),
```

Comment:
word size of `slot0` and `slot1` are almost definitely not the same as word size of `m_main`

Reply:
Fixed